### ISSUE TITLE
Edit Image - the progress bar when image is being saved is confusing

### ISSUE THREAD

**Issue by wwalc** [ckfinder/ckfinder#167]
1. Edit an image stored in a cloud.
2. Crop it and click Save button.

Result:

When image is being downloaded, the progress bar don't even start which suggests that the feature doesn't work:
![screen shot 2015-05-28 at 14 29 44](https://cloud.githubusercontent.com/assets/545175/7860056/1364cc0c-0546-11e5-8d20-51d07e5122ef.png)

When the image is uploaded to the server, the progress bar does not move at all as well.
![screen shot 2015-05-28 at 14 29 53](https://cloud.githubusercontent.com/assets/545175/7860059/1857bc56-0546-11e5-904e-ed7276d8fa20.png)

Basically the progress bar is updated just once after the image is downloaded. It should be updated dynamically while the file is downloaded/uploaded.

**Comment by jodator** [comment]
Fixed. The updated code will be publicly available in the upcoming release.
